OpenRA.Game/Graphics/SheetBuilder.cs

```diff
@@ -86,6 +84,13 @@ public SheetBuilder(SheetType t, Func<Sheet> allocateSheet, int margin = 1)
 		public Sprite Add(byte[] src, SpriteFrameType type, Size size, bool premultiplied = false) { return Add(src, type, size, 0, float3.Zero, premultiplied); }
 		public Sprite Add(byte[] src, SpriteFrameType type, Size size, float zRamp, in float3 spriteOffset, bool premultiplied = false)
 		{
+			if (Current == null)
+			{
+				Current = allocateSheet();
+				Current.CreateBuffer();
```

Comment:
We can get away without the `CreateBuffer` call here and below, they'll be created when `Util.FastCopyIntoChannel` accesses the sheet data anyway.

Reply:
I'm not sure we can. I feel like we need a bit more unified handling of empty sprites

Reply:
Removed Current.CreateBuffer();